### ISSUE TITLE
Backup folder requirement

### ISSUE THREAD

**Issue by Transisto** [Foundation-Devices/passport-firmware#81]
When we go to restore a backup it say there is nothing and that the file should end in .7z

It should also say that the file should be in the /backup/ folder.

**Comment by BitcoinQnA** [comment]
Hi,

Have you verified that there is actually a file on the microSD card? If yes, have you tried removing and re-inserting the microSD into the device?

**Comment by Transisto** [comment]
Yes, I did I the testing that's how I know an additional requirement is to be in a specific folder.

People might move the file around and end up outside of such folder and be confused when importing.
